editor/animation_track_editor.cpp

```diff
@@ -4740,11 +4740,13 @@ void AnimationTrackEditor::_new_track_node_selected(NodePath p_path) {
 
 	switch (adding_track_type) {
 		case Animation::TYPE_VALUE: {
+			prop_selector->set_multiselect(true);
```

Comment:
We have to set_multiselect each time user switches between type of track to add. This is because the same PropertySelector is used for three of the types: (Property, Bezier and Blend)

The other instance of a PropertySelector is method_selector which would default to single on creation and its only used in one place.

Should we consider using different PropertySelector for each animation, or a signal that says when the type of animation has changed for that instance of PropertySelector. Or just keep this approach?